modules/exploits/linux/http/cisco_rv32x_rce.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer::HTML
+  include Msf::Exploit::CmdStager
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Cisco RV320 and RV325 Unauthenticated Remote Code Execution",
+      'Description'    => %q{
+This exploit module combines an information disclosure (CVE-2019-1653)
+and a command injection vulnerability (CVE-2019-1652) together to gain
+unauthenticated remote code execution on Cisco RV320 and RV325 small business
+routers. Can be exploited via the WAN interface of the router. Either via HTTPS
+on port 443 or HTTP on port 8007 on some older firmware versions.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'RedTeam Pentesting GmbH','Benjamin Grap' ],
```

Comment:
Traditionally, the `Author` array makes use of comments for attribution. Suggest something like this:

```ruby
      'Author'         => [
        'RedTeam Pentesting GmbH', # Discovery
        'Benjamin Grap'            # Metasploit
      ],
```